test/functional/Valhalla/src_lw5/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -1386,6 +2031,100 @@ static public void testGCFlattenedMegaObjectArray() throws Throwable {
 		MegaValueObject! value = array[0];
 	}
 
+	/*
+	 * Create large number of value types and instantiate them
+	 *
+	 * value Point2D# {
```

Comment:
I see `#` is added after 2D. Are you using it to indicate the valueIndex ?

Reply:
Its a typo actually